module/behaviour/planning/BezierWalkPathPlanner/src/BezierWalkPathPlanner.cpp

```diff
@@ -342,7 +342,7 @@ namespace module {
                         // emit(graph("forwardSpeed2", finalForwardSpeed));
 
 
-                        emit(std::make_unique<KickPlan>(KickPlan{kick_target, KickType::SCRIPTED}));
+                        // emit(std::make_unique<KickPlan>(KickPlan{kick_target, KickType::SCRIPTED}));
```

Comment:
If it's not needed anymore just delete it, Git diffs are easier to read when things are just removed over commented out. Unless this is something that will be regularly toggled.